sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorAspNetCoreEventSource.cs

```diff
@@ -60,6 +59,15 @@ public void GetEnvironmentVariableFailed(string envVarName, Exception ex)
             }
         }
 
+        [NonEvent]
+        public void MapLogLevelFailed(EventLevel level)
+        {
+            if (IsEnabled(EventLevel.Warning))
+            {
+                MapLogLevelFailed(level.ToString());
+            }
+        }
+
```

Comment:
Please move this method down to line 85, to keep it adjacent to the other method with the same name.